packages/unplugin/src/plugin/core.ts

```diff
@@ -125,27 +129,66 @@ export const unpluginFactory: UnpluginFactory<PandaPluginOptions | undefined> =
   }
 
   let server: ViteDevServer
-  let lastCss: string | undefined
+
+  let latestCss: string | undefined
+  // Necessary because when using virtual:pandacss in cases with many files,
```

Comment:
does that imply that there's no issue when using an actual file ? with `options.outfile`

Reply:
That's right, I didn't see the issue when testing with outfile. I wish I understood better why it's different